docs/troubleshooting.md

```diff
@@ -67,6 +67,43 @@ You can see what features your CPU has with the following.
 cat /proc/cpuinfo| grep flags  | head -1
 ```
 
+## AMD Radeon GPU Support

```

Comment:
It was suggested in Discord (and not required for this PR), but it will be nice to have something like this support matrix easier to see.

Reply:
I'll start putting together a follow up PR to refine the docs.